Makefile

```diff
@@ -27,3 +27,7 @@ test-cov: generate vet
 
 clean:
 	@rm -rf ./reports ./out
+
+fmt:
+	@gofumpt -w .
```

Comment:
Can we add a development section to the readme that specifies which versions of `gofumpt` and `goimports` needs to be installed? Alternatively, we could have a `make setup` target that installs those dependencies.